src/gridtemplates.f

```diff
@@ -252,34 +248,23 @@ module gridtemplates
 
       contains
 
+!>    @brief This function returns the index of specified Grid
```

Comment:
@Hang-Lei-NOAA You don't need the block starting with @brief before each routine. Follow what you did in 'g2grid.f'

Reply:
Thanks, Will remove all.